relnotes/vararg.migration.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.io.console.AppStatus`
+
+  `formatStaticLine` method was changed to use template-based variadic argument
+  list instead of runtime one.
```

Comment:
What kind of change this actually require from the user? It should be called in the same way, shouldn't it?

Reply:
If user code was passing in `__va*`, calling method will need to also be rewritten to use template instead. Direct calls will keep working as-is.

Reply:
I see :+1: